monailabel/interface/app.py

```diff
@@ -155,12 +155,10 @@ def save_label(self, request):
         Returns:
             JSON containing next image and label info
         """
-        # TODO:: Save label, trigger training (if condition is met)
-        labels_dir = os.path.join(self.studies, "newLabelsTr")
-        os.makedirs(labels_dir, exist_ok=True)
-
-        label_file = os.path.join(labels_dir, os.path.basename(request["image"]))
-        shutil.move(request["label"], label_file)
+        
+        label = io.BytesIO(open(request['label'], 'rb').read())
```

Comment:
for now it's ok.. 
endpoint can send the FP for the api to read it from there.. we can avoid multiple writes for the label file

Reply:
Sounds good, I was going to suggest we keep the file in memory if possible so I save it only once here.